### ISSUE TITLE
Loading of external stylesheets not working

### ISSUE THREAD

**Issue by stijnkoopal** [webpack-contrib/polymer-webpack-loader#48]
Hi,

When not using polymer-webpack-loader, we can do the following:

```html
<dom-module id="test">
    <template>
        <link rel="stylesheet" href="node_modules/plottable/plottable.css">
        <div class="plottable">
        </div>
    </template>
</dom-module>
```

However, when using polymer-webpack-loader (together with css-loader) this no longer works. Ie. the styles show up in the bundle, but the div is not styled.

The problem (I think) is that all links are first parsed from the dom-module, no matter the rel or the position in the dom-module. Hence, in the bundle, all styles are `required` before the actual dom-module. Then shadow-dom kicks in on the browser, and the styles are no longer part of the component, so no styling.

One solution would be:
  - If a rel="stylesheet" is found, the parser should yield another stylus component, with the styles from the stylesheet encapsulated in that component.
 - In the component that initially linked to the stylesheet, the following should be added `<style includes="hash-for-stylus-component-or-something">`

Another would be to encapsulate the stylesheet into the <style></style> tag of the linking component. (Would include stylesheet multiple times if referenced multiple times)

**Comment by ChadKillingsworth** [comment]
The fix for this is already in progress and is part of the changes needed for #12

**Comment by bryandcoulter** [comment]
@stijnkoopal Would it be possible for you to send me either a sample repo or the webpack.config.js that you are using to process the example above. I am getting a slightly different result and just want to double check that I am correctly addressing the issue you are having. 
Thanks Bryan

**Comment by stijnkoopal** [comment]
@bryandcoulter sorry, the example I gave wasn't fully correct. There is a layer of indirection in between with a stylus component. Furthermore, it happens when using type="css"

Please have a look here: https://github.com/stijnkoopal/polymer-webpack-css
- The master branch -non-webpack- contains a setup that shows 2 squares, a red one and a green one
- The `webpack-not-working` branch shows only 1 square: the green one. The styles for the red one are not loaded.

**Comment by constantijn** [comment]
For the record, we have an external CSS file (from Plottable: https://github.com/palantir/plottable ) that we'd like to load once in a web component along with some generic things and app specific styling (colors mostly) to allow us to use charts in our app. We'd then like to use that webcomponent as a basis for all our actual chart components.

**Comment by ChadKillingsworth** [comment]
This was fixed by #51
